File: Urnas-y-Bolas-Profe/src/Color.java
import org.fusesource.jansi.Ansi;

public enum Color
{
    AMARILLO(Ansi.Color.YELLOW), AZUL(Ansi.Color.BLUE),
    ROJO(Ansi.Color.RED), VERDE(Ansi.Color.GREEN);

    private Ansi.Color color;

    Color(Ansi.Color color) {
        this.color=color;
    }

    public Ansi.Color getColor() {
        return color;
    }
}

File: Urnas-y-Bolas/src/Color.kt
import org.fusesource.jansi.Ansi

enum class Color(val color: Ansi.Color) {
    AMARILLO(Ansi.Color.YELLOW),
    AZUL(Ansi.Color.BLUE),
    ROJO(Ansi.Color.RED),
    VERDE(Ansi.Color.GREEN);

    // Método para obtener el color asociado
    fun getColor(): Ansi.Color {
        return color
    }
}
